corehq/ex-submodules/casexml/apps/stock/utils.py

```diff
@@ -45,19 +45,33 @@ def state_stock_category(state):
 def get_current_ledger_transactions(case_id):
```

Comment:
my guess is that we may be able to remove this method entirely and update the callers to use states too but that's definitely out of scope